client/src/partials/patient_invoice/registry/templates/action.cell.html

```diff
@@ -23,5 +23,17 @@
         <span class="text-danger"><i class="fa fa-clone"></i> <span translate>FORM.LABELS.CREATE_CREDIT_NOTE</span></span>
       </a>
     </li>
+    <li class="divider"></li>
+    <li>
+      <a ui-sref="patientRegistry({ filters : { display_name : row.entity.patientName }})" data-method="invoiceReceipt" href>
+        <i class="fa fa-user"></i> <span translate>REPORT.VIEW_PATIENT</span>
+      </a>
+    </li>
+    <li>
+      <a data-method="card" ui-sref="cashRegistry({ filters : { identifiers : { invoice_uuid : row.entity.uuid }, display : { invoice_uuid : row.entity.uuid } }})" href>
```

Comment:
The `data-method` does not line up here. This may cause an issue for developers writing tests.